boards/st/stm32h745i_disco/stm32h745i_disco_stm32h745xx_m7.dts

```diff
@@ -153,7 +153,7 @@
 
 	ethernet-phy@0 {
 		compatible = "ethernet-phy";
-		reg = <0x00>;
+		reg = <0x01>;
```

Comment:
`reg` is supposed to match the `@0` in the node name above.
Would you mind fixing it as well ?

Reply:
I wanted to do it before pushing this version but got a compilation error of `eth_stm32_hal` at [line 64](https://github.com/zephyrproject-rtos/zephyr/blob/e7db0f8aff81436ff07f557a5ca8e2daa1789188/drivers/ethernet/eth_stm32_hal.c#L64). I will give it another try.

Reply:
That's probably because `PHY_ADDR` should be set by dt, not Kconfig.